core/dbt/adapters/cache.py

```diff
@@ -1,21 +1,25 @@
 from collections import namedtuple
 import threading
 from copy import deepcopy
+from typing import Optional
+
+from dbt.adapters.cache import _CachedRelation
```

Comment:
I assume this is so `def add_reference(self, referrer: _CachedRelation):` works inside its own class definition. Instead, let's just use `def add_reference(self, referrer: '_CachedRelation'):`.

Reply:
Good catch, I've updated the code.

Reply:
Don't forget to remove this line, too - it's going to make importing it with python fail!

Reply:
Thanks!